chain/db/issue.go

```diff
@@ -0,0 +1,149 @@
+package db
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"github.com/nknorg/nkn/common"
+	"github.com/nknorg/nkn/common/serialization"
+)
+
+type issueAsset struct {
+	name        string
+	symbol      string
+	totalSupply common.Fixed64
+	precision   byte
+	owner       common.Uint160
+}
+
+func (ia *issueAsset) Empty() bool {
+	return len(ia.name) == 0 && len(ia.symbol) == 0
```

Comment:
Should we also check other fields in `Empty()`?

Reply:
0 total supply and  0 precision is legal.

Reply:
But is it valid that a asset has empty name and symbol?

Reply:
no, asset must have name and symbol.